dev-libs/icu/icu-62.2.recipe

```diff
@@ -0,0 +1,183 @@
+COPYRIGHT="1995-2020 IBM Corporation and others."
+LICENSE="ICU"
+REVISION="1"
+SOURCE_URI="https://github.com/unicode-org/icu/releases/download/release-62-2/icu4c-62_2-src.tgz"
+SOURCE_URI_2="https://github.com/unicode-org/icu/releases/download/release-62-2/icu4c-62_2-data.zip#noarchive"
+SOURCE_URI_3="http://www.iana.org/time-zones/repository/releases/tzdata2020a.tar.gz#noarchive"
+CHECKSUM_SHA256="f51976e6893df7b08e594fa798196f80cdf07df885fec129f55f7afe273b6901"
+CHECKSUM_SHA256_2="1d91357378a35a2acaedfb0e71fac6e6fedd360d145b5280428e2d9b6f0a446a"
+CHECKSUM_SHA256_3="547161eca24d344e0b5f96aff6a76b454da295dc14ed4ca50c2355043fb899a2"
+SOURCE_DIR="icu"
+
+ARCHITECTURES="!x86_gcc2 ?x86_64 ?arm ?sparc"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	icu${secondaryArchSuffix} = $portVersion compat >= 60
+	lib:libicudata$secondaryArchSuffix = $portVersion compat >= 60
+	lib:libicui18n$secondaryArchSuffix = $portVersion compat >= 60
+	lib:libicuio$secondaryArchSuffix = $portVersion compat >= 60
+	lib:libicutest$secondaryArchSuffix = $portVersion compat >= 60
+	lib:libicutu$secondaryArchSuffix = $portVersion compat >= 60
+	lib:libicuuc$secondaryArchSuffix = $portVersion compat >= 60
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	icu${secondaryArchSuffix}_devel = $portVersion
+	cmd:icu_config${secondaryArchSuffix} = $portVersion compat >= 60
+	devel:libicudata${secondaryArchSuffix} = $portVersion compat >= 60
+	devel:libicui18n${secondaryArchSuffix} = $portVersion compat >= 60
+	devel:libicuio${secondaryArchSuffix} = $portVersion compat >= 60
+	devel:libicutest${secondaryArchSuffix} = $portVersion compat >= 60
+	devel:libicutu${secondaryArchSuffix} = $portVersion compat >= 60
+	devel:libicuuc${secondaryArchSuffix} = $portVersion compat >= 60
+	"
+REQUIRES_devel="
+		icu${secondaryArchSuffix} == $portVersion base
+	"
+
+if [ -z "$secondaryArchSuffix" ]; then
+	SUMMARY_tools="The ICU support tools"
+	PROVIDES_tools="
+		icu66_tools = $portVersion
+		cmd:derb
+		cmd:escapesrc
+		cmd:genbrk
+		cmd:genccode
+		cmd:gencfu
+		cmd:gencmn
+		cmd:gencnval
+		cmd:gendict
+		cmd:gennorm2
+		cmd:genrb
+		cmd:gensprep
+		cmd:icuinfo
+		cmd:icupkg
+		cmd:makeconv
+		cmd:pkgdata
+		"
+	REQUIRES_tools="
+		icu${secondaryArchSuffix} == $portVersion base
+		haiku
+		"
+fi
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python3
+	cmd:sed
+	cmd:unzip
+	cmd:tar
+	cmd:gunzip
+	"
+
+BUILD()
+{
+	cd source
+
+	# Created during build and prevent rebuilds.
+	#rm -rf tools/tzcode/vanguard
+	#cp ../../../sources-3/tzdata*.tar.gz tools/tzcode/
+
+	unzip -oq ../../../sources-2/icu4c-*-data.zip
+	rm -f data/in/icudt??l.dat
+	autoconf
+	runConfigure ./configure \
+		--disable-samples --disable-extras --with-data-packaging=archive \
+		--disable-renaming --disable-layoutex
```

Comment:
`--disable-renaming --disable-layoutex` seems to be a new addition to the recipe. Would be nice to know what they do and why they required.

Reply:
Those are optional.